benchmarks/benchmark_sbi/benchmark_meta.json

```diff
@@ -0,0 +1 @@
+{"pretty_name": "sbi: maximum likelihood on test set"}
```

Comment:
can you update the name of the benchmark to something easier to understand ? this is the name that will appear on the front page

Reply:
Yes, maybe just "Simulation-Based Inference"? How do I change that on this PR?